### DISCUSSION TITLE
Still deployment issues

### DISCUSSION THREAD

**Discussion by salvatoreloguercio** [alshedivat/al-folio#370] [Q&A]
Hello, I've been trying to use al-folio for a personal github page - I've been reading the getting started section and the issue 209 and they suggest different strategies: issue 209 suggests to rename master to source and then set github Pages source to master, whereas in the Getting Started guide there is no renaming, just use 'master' and then set Pages source to 'gh-pages'.

Which procedure would be the right one?

Thanks!

**Comment by JulianStreyczek** [comment]
Hello :) You can just edit the `master` branch and use the `gh-pages` branch for deployment, as described in the Getting Started guide. 

(Issue #209 referred to an older version of the project. The branch structure was recently simplified in line with Issue #342 and Pull request #344)
